translations/de/hints/bryukh.html

```diff
@@ -1,40 +1,43 @@
 <body data-author="bryukh">
 <div class="question-answer">
-    <div class="question">
-        I have no idea how to start solving this mission
-    </div>
-    <div class="answer">
-        To iterate indexes in an array use "<a href="https://docs.python.org/2/library/functions.html#range">range</a>".
-<pre class="brush: python">
+  <div class="question">
+    Ich habe keine Ahnung wo ich mit der Lösung anfangen soll.
+  </div>
+  <div class="answer">
+    Um über Indizes eines Arrays zu iterieren, kannst Du die eingebaute Funktion
+    "<a href="https://docs.python.org/3.6/library/stdtypes.html#ranges">range</a>"
+    verwenden.
```

Comment:
ist ein überbleibsel und Python 2 wird auch gar nicht mehr unterstützt.
Die Änderung für die Links in alles Sprachversionen ist schon auf dem Zettel.